README.md

```diff
@@ -460,7 +460,7 @@ when creating the S3 bucket or DynamoDB table.
 
 ### Keep your CLI flags DRY
 
-#### Motivation
+#### <a name="extra_arguments"></a>Motivation
```

Comment:
GitHub headers automatically have anchor links. You could just link to #keep-your-cli-flags-dry.